### PR TITLE
package/linux-mainline: Bump to 5.19

### PR DESCRIPTION
**Pull request by alistair23**
Bump from the 5.18 to the 5.19 kernel. No rM2 specific changes here, just keeping up to date with the Linux kernel

### CODE REVIEW

File: package/linux-mainline/package [hunk 1]
```diff
@@ -6,7 +6,7 @@ archs=(rm2)
 pkgnames=(linux-mainline)
 pkgdesc="reMarkable 2 kernel based on the mainline kernel"
 url=https://www.kernel.org
-pkgver=5.18.0-1
+pkgver=5.19.0-0
```
Comment (gbyl) [inline package/linux-mainline/package]:
```suggestion
pkgver=5.19.0-1
```

toltec package [revision versions](https://github.com/toltec-dev/build/blob/main/docs/recipe-format.md#pkgver-field) start at 1

**Review by gbyl — CHANGES_REQUESTED** [review]

**Comment by Etn40ff** [comment]
Could you please also change the commands to package the resulting kernel from 
```
tar --owner root:0 --group root:0 --mtime=$timestamp \
    -cjf "$pkgdir"/opt/usr/share/kernelctl/"$archive" -C "$staging" .
```
to
```
(cd "$staging" && tar --owner root:0 --group root:0 --mtime=$timestamp \
    -cjf "$pkgdir"/opt/usr/share/kernelctl/"$archive" boot/* lib/modules/*)
```

**Comment by Etn40ff** [comment]
LGTM

**Comment by alistair23** [comment]
Ping!

**Comment by alistair23** [comment]
Ping!

**Comment by Eeems** [comment]
@Etn40ff Did you test this on device?

**Comment by Etn40ff** [comment]
> @Etn40ff Did you test this on device?

Yes, back in August after the changes I asked for. If I remember correctly it worked as expected.

**Comment by Eeems** [comment]
@Etn40ff can you do a quick test on device since we've changed the package. I just need to know the following:

- Install works fine and has the expected output
- Using kernelctl to change the kernel to this, and then to something else works
- Uninstall works fine

**Comment by Etn40ff** [comment]
All seems to be fine on my end. One last change that needs to happen is that `postremove` must be deleted. Other than that it has my green light.

**Comment by Etn40ff** [comment]
Tested again. LGTM

**Review by Eeems — APPROVED** [review]

**Pull request merged**